pennylane/templates/subroutines/arbitrary_unitary.py

```diff
@@ -83,7 +83,8 @@ class ArbitraryUnitary(Operation):
     .. code-block:: python
 
         def arbitrary_nearest_neighbour_interaction(weights, wires):
-            qml.broadcast(unitary=ArbitraryUnitary, pattern="double", wires=wires, parameters=weights)
+            for w in range(0, len(wires) - 1, 2):
+                ArbitraryUnitary(weights, wires=[w, w + 1])
```

Comment:
Do we need to index through the `weights` here, like so?
```suggestion
                ArbitraryUnitary(weights[w], wires=[w, w + 1])
```

Reply:
`w` increments by a stride of 2, so we need different indexing, but nice catch!